src/components/network/network-table.js

```diff
@@ -50,10 +71,74 @@ genotet.NetworkTable.prototype.create = function(table, edges) {
       {title: 'Weight', data: 'weight'},
       {title: '', data: 'added'}
     ],
+    select: true,
+    dom: '<"row"<"col-sm-12"B>>' +
+      '<"row"<"col-sm-5"l><"col-sm-7"f>>' +
+      '<"row"<"col-sm-12"tr>>' +
+      '<"row"<"col-sm-5"i><"col-sm-7"p>>',
+    buttons: [
```

Comment:
The dataTables supports multiple selects. But the buttons seem to only handle single edge addition/removal.